cleverhans/attacks.py

```diff
@@ -1527,7 +1530,7 @@ def generate(self, x, g, **kwargs):
         # Parse and save attack-specific parameters
         assert self.parse_params(**kwargs)
 
-        g_feat = self.model.get_layer(g, self.layer)
```

Comment:
Is there a particular reason you removed the `get_layer` method from the `Model` class? I would rather keep it to maintain backward compatibility unless it is incompatible with the new code you added.

Reply:
It's not used anywhere as far as I can tell, it seems it was something inherited from Keras.
I'll put them back in for backward compatibility.

Reply:
I'd prefer to keep it too.

In my experimental directory at Google, it looks like I'm using this function about 50 times. So while it's not used within the library, it is used by users of the library, or at least it is if I'm a representative user.

Reply:
+1  (I also use that function a lot)

Reply:
Okay done.

Reply:
Put it back, but extended it a bit to take **kwargs, it's still backward compatible but more flexible to pass arguments to fprop.